modules/renderer/src/renderer.ts

```diff
@@ -137,7 +137,7 @@ export default class Renderer extends EventSource {
     }
   }
 
-  setRenderOptions(options: LayoutOptions) {
+  setLayoutOptions(options: LayoutOptions) {
```

Comment:
Not sure why you feel that it must be renamed, but I find the names `LayoutOptions` vs. `LayoutSettings` super confusing. It was called `RenderOptions` before because there could be renderer-specific settings in there in the future (e.g. highlight colors). If you have to rename it I propose you name it `setOptions`.

Reply:
Layout means for laying out the graph. Rendering options do not change, in my opinion, if we change the layout method. I can rename it to setOptions, but it does not seem specific enough.

Reply:
Actually, the font has something to do with rendering, changing to setOptions